src/utils/functions/program-section-template-validation/programSectionTemplateValidation.ts

```
@@ -0,0 +1,59 @@
+    groupedTypes: ContentType[],
+): ProgramSectionContent[] => {
+    const relevant = contents.filter(
+        (c) => groupedTypes.includes(c.contentType) && c.groupIndex !== null && c.groupIndex !== undefined,
+    );
+
+    const uniqueSorted = Array.from(new Set(relevant.map((c) => c.groupIndex as number))).sort((a, b) => a - b);
+
+    if (uniqueSorted.length === 0) return contents;
+
+    const map = new Map<number, number>();
+    uniqueSorted.forEach((oldIdx, newIdx) => map.set(oldIdx, newIdx));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)